niworkflows/anat/ants.py

```diff
@@ -0,0 +1,559 @@
+    # ImageMath ${DIMENSION} ${EXTRACTION_MASK} ME ${EXTRACTION_MASK} 5
+    me_7_2 = pe.Node(ImageMath(operation='ME', op2='5'), name='22_me_7_2')
+
+    # De-pad
+    depad_mask = pe.Node(ImageMath(operation='PadImage', op2='-%d' % padding),
+                         name='23_depad_mask')
+    depad_segm = pe.Node(ImageMath(operation='PadImage', op2='-%d' % padding),
+                         name='24_depad_segm')
+    depad_gm = pe.Node(ImageMath(operation='PadImage', op2='-%d' % padding),
+                       name='25_depad_gm')
+    depad_wm = pe.Node(ImageMath(operation='PadImage', op2='-%d' % padding),
+                       name='26_depad_wm')
+    depad_csf = pe.Node(ImageMath(operation='PadImage', op2='-%d' % padding),
+                        name='27_depad_csf')
+    wf.connect([
+        (inputnode, pad_mask, [('in_mask', 'op1')]),
+        (inputnode, atropos, [('in_files', 'intensity_images'),
+                              ('in_mask', 'mask_image')]),
+        (atropos, pad_segm, [('classified_image', 'op1')]),
+        (pad_segm, sel_labels, [('output_image', 'in_segm')]),
+        (sel_labels, get_wm, [('out_wm', 'op1')]),
+        (sel_labels, get_gm, [('out_gm', 'op1')]),
+        (get_gm, fill_gm, [('output_image', 'op1')]),
+        (get_gm, mult_gm, [('output_image', 'first_input'),
+                           (('output_image', _gen_name), 'output_product_image')]),
+        (fill_gm, mult_gm, [('output_image', 'second_input')]),
+        (get_wm, relabel_wm, [('output_image', 'first_input'),
+                              (('output_image', _gen_name), 'output_product_image')]),
+        (sel_labels, me_csf, [('out_csf', 'op1')]),
+        (mult_gm, add_gm, [('output_product_image', 'op1')]),
+        (me_csf, add_gm, [('output_image', 'op2')]),
+        (add_gm, relabel_gm, [('output_image', 'first_input'),
+                              (('output_image', _gen_name), 'output_product_image')]),
+        (relabel_wm, add_gm_wm, [('output_product_image', 'op1')]),
+        (relabel_gm, add_gm_wm, [('output_product_image', 'op2')]),
+        (add_gm_wm, sel_labels2, [('output_image', 'in_segm')]),
+        (sel_labels2, add_7, [('out_wm', 'op1'),
+                              ('out_gm', 'op2')]),
+        (add_7, me_7, [('output_image', 'op1')]),
+        (me_7, comp_7, [('output_image', 'op1')]),
+        (comp_7, md_7, [('output_image', 'op1')]),
+        (md_7, fill_7, [('output_image', 'op1')]),
+        (fill_7, add_7_2, [('output_image', 'op1')]),
+        (pad_mask, add_7_2, [('output_image', 'op2')]),
+        (add_7_2, md_7_2, [('output_image', 'op1')]),
+        (md_7_2, me_7_2, [('output_image', 'op1')]),
+        (me_7_2, depad_mask, [('output_image', 'op1')]),
+        (add_gm_wm, depad_segm, [('output_image', 'op1')]),
+        (relabel_wm, depad_wm, [('output_product_image', 'op1')]),
+        (relabel_gm, depad_gm, [('output_product_image', 'op1')]),
+        (sel_labels, depad_csf, [('out_csf', 'op1')]),
```

Comment:
Outputs of `depad_segm`, `depad_wm`, `depad_gm`, `depad_csf` are unused. Are these supposed to be in `outputnode.out_segm` and `outputnode.out_tpms`?

Reply:
They were missing, you are right.